discord/message_update_builder.go

```diff
@@ -254,6 +254,20 @@ func (b *MessageUpdateBuilder) SetSuppressEmbeds(suppressEmbeds bool) *MessageUp
 	return b
 }
 
+// SetIsComponentsV2 adds/removes discord.MessageFlagIsComponentsV2 to the Message flags. Once the message has been sent, the flag cannot be removed from the message when editing the message
```

Comment:
```suggestion
// SetIsComponentsV2 adds/removes discord.MessageFlagIsComponentsV2 to the Message flags. Once a message with the flag has been sent, it cannot be removed by editing the message.
```

could you also add this notice to the `discord.MessageFlagIsComponentsV2` & other `SetIsComponentsV2` methods?
The `WebhookMessageUpdateBuilder` seems to also miss this method